ugs-pendant/src/main/java/com/willwinder/universalgcodesender/pendantui/PendantUI.java

```diff
@@ -219,11 +239,49 @@ public void handle(String target,Request baseRequest,HttpServletRequest request,
         @Override
         public void handle(String target,Request baseRequest,HttpServletRequest request,HttpServletResponse response) throws IOException, ServletException {
             baseRequest.setHandled(true);
-            mainWindow.updateSystemState(systemState);
+            updateSystemState(systemState);
             response.getWriter().print(getSystemStateJson());
         }
     }
-    
+
```

Comment:
I think this can be marked as `@deprecated` with a comment that it is only used by the old pendant

Reply:
Done. I think that these Handlers should be deprecated and removed once we get the new Pendant verified by the users.